static/app/utils/regions/index.tsx

```diff
@@ -0,0 +1,91 @@
+import ConfigStore from 'sentry/stores/configStore';
+import {Organization, Region} from 'sentry/types';
+
+export enum RegionDisplayName {
+  US = 'United States of America (US)',
+  DE = 'European Union (EU)',
+}
+
+export enum RegionFlagIndicator {
+  US = '🇺🇸',
+  DE = '🇪🇺',
+}
+
+export interface RegionData {
+  regionDisplayName: string;
+  regionName: string;
+  regionUrl: string;
+  regionFlag?: RegionFlagIndicator;
+}
```

Comment:
do they all need to be prefixed with `region` if the type is already `RegionData`?